1-exercises/A-undefined/exercise.js

```diff
@@ -10,27 +10,29 @@
 */
 
 // Example 1
-let a;
+let a = 1;
```

Comment:
Dear Jud, the question was why is the given expression undefined so, you are expected to explain. for instance, I said that no value was assigned to the variable when declaring.